src/main/scala/com/atomist/rug/kind/core/ProjectMutableView.scala

```diff
@@ -428,14 +429,23 @@ class ProjectMutableView(
           case wtf =>
             throw new RugRuntimeException(ed.name, s"Unexpected editor failure: $wtf", null)
         }
-        case _ => throw new MissingRugException(s"Unable to find editor: [$editorName]")
+        case None =>
+          if(editorName.contains(":")){
+            val shortName = StringUtils.substringAfterLast(editorName, ":")
+            if(rug.findRug(shortName).nonEmpty){
+              throw new EditorNotFoundException(s"Could not find editor: $editorName. Did you mean: $shortName?")
```

Comment:
I guess, but we can't put too information in exceptions IMO. This is a helpful error message, but does not provide documentation.